build/install.sh

```diff
@@ -21,7 +21,7 @@ if [[ $PYTHON == "NOT_SET" ]]; then
 fi
 
 PYTHON_VERSION=$($PYTHON -V 2>&1 | grep -Eo '([0-9]+\.[0-9]+\.[0-9]+)$')
-if [[ ${PYTHON_VERSION} < "3.7.0" ]]; then
+if [[ ${PYTHON_VERSION} < "3.7.0" && ${PYTHON_VERSION} < "3.10.0" ]]; then
```

Comment:
Unfortunately, this won't be true for python versions lower than 3.7.0

```shell
$ cat test2.sh 
#!/usr/bin/env bash

if [[ $1 < "3.7.0" && $1 < "3.10.0" ]]; then
  echo "[install] Python version 3.7.0 or higher is required."
  exit 1
fi
$ ./test2.sh 3.7.0
$ ./test2.sh 3.10.0
$ ./test2.sh 3.5.0
$ ./test2.sh 3.3.0
```

I guess comparing semver strings is not an option anymore. That being said, I found a simple [bash method](https://github.com/mritd/shell_scripts/blob/master/version.sh) that might be useful here.

Example:
```shell
$ cat test.sh 
#!/usr/bin/env bash

function version_lt() { test "$(echo "$1 3.7.0" | tr " " "\n" | sort -rV | head -n 1)" != "$1"; }

if version_lt $1 "3.7.0"; then
    echo "$1 is les than 3.7.0"
fi
$ ./test.sh 3.10.0
$ ./test.sh 3.12.0
$ ./test.sh 3.7.0
$ ./test.sh 3.6.99
3.6.99 is les than 3.7.0
$ ./test.sh 3.5.0
3.5.0 is les than 3.7.0
$ ./test.sh 3.5.50
3.5.50 is les than 3.7.0
```

Or maybe do you have any other ideas? I'm open to suggestions.

Finally, identical version checking is in cookiecuter template `build/install.sh` and should be fixed as well.
https://github.com/rszamszur/fastapi-mvc/blob/acdde1d7184a200846fedd093c84d06fae4f2b56/fastapi_mvc/template/%7B%7Bcookiecutter.folder_name%7D%7D/build/install.sh#L23-L27

Reply:
Oops, my bad!
I think I found a more convenient way.
You're welcome! Thank you for this very convenient tool!